### PR TITLE
chore: update command-line.md

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: docs/topics/command-line.md [hunk 1]
```diff
@@ -19,7 +22,7 @@ To install the Kotlin compiler manually:
 2. Unzip the standalone compiler into a directory and optionally add the `bin` directory to the system path.
 The `bin` directory contains the scripts needed to compile and run Kotlin on Windows, macOS, and Linux.
 
-> For Windows users who want to use the command-line compiler, we recommend using the manual installation method.
+> If you want to use the Kotlin command-line compiler on Windows, we recommend using the manual installation method.
```
Comment (zamulla) [inline docs/topics/command-line.md:25]:
```suggestion
> If you want to use the Kotlin command-line compiler on Windows, we recommend installing it manually.
```

File: docs/topics/command-line.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
 
->Installing the command-line compiler is not an essential step to use Kotlin. A general way to write Kotlin applications is using an
->IDE - [IntelliJ IDEA](https://www.jetbrains.com/idea/) or [Android Studio](https://developer.android.com/studio).
->They provide full Kotlin support out of the box without needing additional components. Learn how to 
-[get started with Kotlin in an IDE](getting-started.md).
+> Installing the command-line compiler is not an essential step to use Kotlin.
+> The common approach is to write Kotlin applications using IDEs or code editors with official Kotlin support,
+> such as [IntelliJ IDEA](https://www.jetbrains.com/idea/), [JetBrains Fleet](https://www.jetbrains.com/fleet/),
+> or [Android Studio](https://developer.android.com/studio).
+> These provide full Kotlin support right out of the box, no extra components needed.
```
Comment (zamulla) [inline docs/topics/command-line.md:9]:
```suggestion
> They provide full Kotlin support right out of the box, no extra components needed.
```

I feel that "they" is a clearer word here, "these" more naturally refers to something perceived visually.

File: docs/topics/command-line.md [hunk 3]
```diff
@@ -133,9 +137,10 @@ kotlinc -script list_folders.kts -- -d <path_to_folder_to_inspect>
 ```
 
 Kotlin provides experimental support for script customization, such as adding external properties,
-providing static or dynamic dependencies, and so on. Customizations are defined by so-called *Script definitions* -
-annotated kotlin classes with the appropriate support code. The script filename extension is used to select the appropriate
-definition. Learn more about [Kotlin custom scripting](custom-script-deps-tutorial.md).
+providing static or dynamic dependencies, and so on.
+Customizations are defined by so-called _Script definitions_ – annotated kotlin classes with the appropriate support code.
```
Comment (zamulla) [inline docs/topics/command-line.md:141]: Should "Script definitions" be capitalized here? On the page linked it looks like a simple term.

**Review by zamulla — APPROVED** [review]
Only the minorest suggestions.

File: docs/topics/command-line.md (hunk 3 above)
Comment (koshachy) [inline docs/topics/command-line.md:141]:
```suggestion
Customizations are defined by so-called _script definitions_ – annotated kotlin classes with the appropriate support code.
```

Comment (koshachy) [reply docs/topics/command-line.md]: Updated!

**Pull request merged**